### DISCUSSION TITLE
DARK MODE PLEASE

### DISCUSSION THREAD

**Discussion by om3ga6400** [yhype/yhype-feedback#16] [General]
PLEASE ADD DARK MODE MY EYES ARE FEELING LIKE THE SURFACE OF THE SUN!!!!!!!!

**Comment by r2hu1** [comment]
i can code it but the source is private or hidden

**Reply by om3ga6400** [reply]
i went blind a LONG time ago buddy

**Reply by antonkomarev** [reply]
@om3ga6400 @r2hu1

You can try new interface, it includes dark theme and has mobile support now!

<img width="1511" height="829" alt="image" src="https://github.com/user-attachments/assets/22065a8d-688e-4184-b03e-00bfb2766bec" />

**Reply by om3ga6400** [reply]
only took 4 years for dark mode

**Reply by r2hu1** [reply]
> only took 4 years for dark mode

lol

**Reply by antonkomarev** [reply]
@om3ga6400 it was too hard to stop hating javascript and start loving typescript :)
